common/src/main/java/io/pravega/common/io/serialization/VersionedSerializer.java

```diff
@@ -475,7 +475,7 @@ void deserializeContents(InputStream stream, ReaderType target) throws IOExcepti
      *
      * Example:
      * <pre>
-     *
+     * {@code
```

Comment:
@sachin-j-joshi removed this in #3116 to reduce warnings. Why are you adding it back?

Reply:
The @code issue causes warnings when javadoc can't distinguish between the code annotations and its own annotations in java 8. There is a workaround I found where the code snippet can be modified like this:
<pre><code>
...... your code snippet here with
     {@literal @}your annotation here
''
</code><pre>
Verified it does what you want in javadoc. Before it was truncating the code segment. Literal tells it to take it literally and not interpret it as a javadoc annotation.

Reply:
hmmm..... did that truncate my example fix?